### PR TITLE
[Estuary] sync

### PR DESCRIPTION
**Pull request by phil65**
@ronie

### CODE REVIEW

**Comment by ronie** [comment]
the DialogPVRInfo.xml fix got reverted here: 
https://github.com/phil65/skin.estuary/commit/2d8285ba0fca6e86094ec8be899b9738d873e98c

looks ok for the rest.

**Comment by jjd-uk** [comment]
Did this get deliberately left out for v18 only? https://github.com/phil65/skin.estuary/commit/6d7939b44bd587ee5ad24f8cf5d1aceec1640f95

**Comment by phil65** [comment]
yup.

**Comment by jjd-uk** [comment]
Ok just wanted check.

**Comment by MartijnKaijser** [comment]
@phil65 could you update the PR so we can merge it?

**Comment by phil65** [comment]
no need to update anything I think? ronie pointed out a commit in my repo, not related to this PR.

**Comment by ronie** [comment]
it was supposed to be included :-)

let me try again: https://github.com/xbmc/xbmc/pull/11724
